packages/fetch-runtime/index.js

```diff
@@ -1,3 +1,4 @@
+require('abortcontroller-polyfill/dist/abortcontroller-polyfill-only');
```

Comment:
This should probably come after `fetch` polyfill. See here: `https://github.com/mo/abortcontroller-polyfill/blob/master/src/polyfill.js#L26`

Reply:
Good point, changed the order.